app/models/todo.rb

```diff
@@ -1,2 +1,5 @@
 class Todo < ActiveRecord::Base
+  def as_json
+    { id: id, title: title, is_completed: is_completed }
+  end
```

Comment:
This is fine for now. But I'm sure Annie will be showing you [Active Model Serializers](https://github.com/rails-api/active_model_serializers) at some point.

Don't change anything yet! But rest assured that there is an even cooler way to do this